tests/test_tutorial/test_request_files/test_tutorial003.py

```diff
@@ -1,11 +1,45 @@
+import importlib
+
+import pytest
+from fastapi import FastAPI
 from fastapi.testclient import TestClient
 
-from docs_src.request_files.tutorial003 import app
+from ...utils import needs_py39
+
+
+@pytest.fixture(
+    name="app",
+    params=[
+        "tutorial003",
+        "tutorial003_an",
+        pytest.param("tutorial003_py39", marks=needs_py39),
+        pytest.param("tutorial003_an_py39", marks=needs_py39),
+    ],
+)
+def get_app(request: pytest.FixtureRequest):
+    mod = importlib.import_module(f"docs_src.request_files.{request.param}")
+
+    return mod.app
+
+
+@pytest.fixture(name="client")
+def get_client(app: FastAPI):
+    client = TestClient(app)
+    return client
+
 
-client = TestClient(app)
+file_required = {
+    "detail": [
+        {
+            "loc": ["body", "files"],
+            "msg": "field required",
+            "type": "value_error.missing",
+        }
+    ]
+}
```

Comment:
```suggestion
```

Is this needed? :thinking: 

I suspect this was copy-pasted from the other file and it was no longer needed there either. :sweat_smile: 

But in that case, we could remove it now. :fire: